src/serverExplorer.ts

```diff
@@ -950,7 +950,14 @@ export class ServerExplorer implements TreeDataProvider<RSPState | ServerStateNo
             return Array.from(this.RSPServersStatus.values()).map(rsp => rsp.state);
         } else if (this.isRSPElement(element) && (element as RSPState).serverStates !== undefined) {
             // rsp parent -> return servers
-            return (element as RSPState).serverStates;
+            const zzz = 33;
+            const abc = zzz+212;
+            const qqq = abc+3;
+            console.log(qqq);
+            const tmp = (element as RSPState).serverStates;
+            const cloned = [...tmp];
+            cloned.sort((a,b) => (a.server.id.localeCompare(b.server.id)));
```

Comment:
While already having left a comment above, I quickly add another one here, I hope you won't mind. :)

I'm not sure how pedant you are about keeping conventions in this project, but this would be more in-line with the current code base:

```typescript
cloned.sort((a, b) => a.server.id.localeCompare(b.server.id));
```